.travis.yml

```diff
@@ -15,33 +15,31 @@ env:
    - TVOS_SCHEME="IGListKit-tvOS"
    - MACOS_SCHEME="IGListKit-macOS"
 
-   - IOS_SDK=iphonesimulator10.3
-   - TVOS_SDK=appletvsimulator10.2
-   - MACOS_SDK=macosx10.12
+   - IOS_SDK=iphonesimulator11.2
+   - MACOS_SDK=macosx10.13
+   - TVOS_SDK=appletvsimulator11.2
 
    - IOS_EXAMPLE_WORKSPACE="Examples/Examples-iOS/IGListKitExamples.xcworkspace"
    - TVOS_EXAMPLE_WORKSPACE="Examples/Examples-tvOS/IGListKitExamples.xcworkspace"
    - MACOS_EXAMPLE_WORKSPACE="Examples/Examples-macOS/IGListKitExamples.xcworkspace"
    - EXAMPLE_SCHEME="IGListKitExamples"
 
    matrix:
-   - DESTINATION="OS=8.1,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="YES" CHECK_MARKDOWN="YES"
-   - DESTINATION="OS=8.2,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=8.3,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=8.4,name=iPhone 6" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-
-   - DESTINATION="OS=9.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=9.1,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=9.2,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO"  BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=9.3,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="NO" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-
-   - DESTINATION="OS=10.0,name=iPhone 6s"  SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=10.1,name=iPhone 6s"  SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=10.2,name=iPad Air 2" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=10.3.1,name=iPad Air 2" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
-
-   - DESTINATION="OS=9.0,name=Apple TV 1080p"  SDK="$TVOS_SDK" SCHEME="$TVOS_SCHEME" RUN_TESTS="NO"  BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
-   - DESTINATION="OS=10.0,name=Apple TV 1080p" SDK="$TVOS_SDK" SCHEME="$TVOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
+   - DESTINATION="OS=9.0,name=iPhone 6 Plus"  SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="YES"
```

Comment:
shouldn't have copied that other `.travis.yml` wholesale 😅 

- missing `RUN_UI_TESTS`
- missing `CHECK_MARKDOWN` (on first entry)
- missing iOS 8 tests (my library dropped iOS 8 support)

Reply:
loool dammit, ok updating